### DISCUSSION TITLE
LocalHost is not updated

### DISCUSSION THREAD

**Discussion by vinzenzgeisler** [vercel/next.js#24602] [Help]
If I change the index.js file and then save it, the localhost is not updated. Can someone help me?

**Comment by bobarros** [comment]
Did you just update WSL 1 to WSL 2?

**Reply by vinzenzgeisler** [reply]
Thanks for your answer!
I have laboriously downloaded WSL2 and want to enter the following command in Ubuntu:

npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn-starter/tree/master/learn-starter"

The answer is:

bash: / mnt / d / programming / Node.
js / npx: / bin / sh ^ M: bad interpreter: No such file or directory

Can you help me again

**Reply by bobarros** [reply]
The first thing is that if you use WSL2, you can't have auto-refresh after saving a file using the Windows filesystem.
About your second question, what happens when you type only 'npx'? And when you try just 'node -v'?

**Reply by vinzenzgeisler** [reply]
Okay, I'm trying to understand the whole thing.

When I type 'npx' in Ubuntu it comes:
-bash: /mnt/d/Programmieren/Node.js/npx: / bin / sh ^ M: bad interpreter: No such file or directory

When I enter 'node-v' comes:
node-v: command not found

![image](https://user-images.githubusercontent.com/82551070/116791311-b195fa80-aab9-11eb-95fe-0639ba2bfdab.png)

**Comment by bobarros** [comment]
It's not node-v, is node(space)-v. But I think my theory was right, since npx comes with node.

You have a brand new ubuntu environment that you need to install Node.

**Reply by vinzenzgeisler** [reply]
I already thought that it was with the space but the same thing happened
![image](https://user-images.githubusercontent.com/82551070/116791479-f0788000-aaba-11eb-90ee-83f514602fc3.png)

**Comment by bobarros** [comment]
I told you in the previous comment. If you don't have npx, you don't have node.

You need to install node in your ubuntu environment.
